loader/base_loader.py

```diff
@@ -54,10 +54,12 @@ def adjust_time(self, time):
         tc_offset = datetime.now(pytz.timezone(self.time_zone)).utcoffset()
         return time + tc_offset
 
-    @abstractclassmethod
+    @classmethod
```

Comment:
no need to add this？

Reply:
It seems necessary because document says that
```
the classmethod() decorator is now correctly identified as abstract when applied to an abstract method.
```

Reply:
Can you test?  I will test now.

Reply:
tested. Only  abstractmethod should be ok, Can you also test both of them?

Reply:
`get_all_track_data ` is used as an instance method, so I think it's ok to remove it.
https://github.com/yihong0618/GitHubPoster/blob/840644f6018f1153e546838994f49ffd3e0dae77/cli.py#L377